hvac/tests/test_integration.py

```diff
@@ -531,3 +531,39 @@ def test_wrapped_token_revoke(self):
         # Attempt to validate token
         with self.assertRaises(exceptions.Forbidden):
             lookup = self.client.lookup_token(result['auth']['client_token'])
+
+    def test_create_token_explicit_max_ttl(self):
+
+        token = self.client.create_token(ttl='30m', explicit_max_ttl='5m')
+
+        assert token['auth']['client_token']
+
+        assert token['auth']['lease_duration'] == 300
```

Comment:
`lease_duration` returns time in seconds this line should be `assert token['auth']['lease_duration'] == 1800`